src/current/v25.1/time.md

```diff
@@ -76,7 +76,7 @@ You can use an [`ALTER COLUMN ... SET DATA TYPE`]({% link {{ page.version.versio
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> CREATE TABLE time (time_id INT PRIMARY KEY, time_val TIME);
```

Comment:
Related to this is the behaviour with decimal and timestamp columns. You can reduce the precision and you will lose precision in the column values. This is the same as the postgres implementation. More details can be seen in this issue: https://github.com/cockroachdb/cockroach/issues/134711. Not sure if you want to call this out or not.

Reply:
oh nice! i have added back the deleted stuff about reducing precision and modified it to go the other way now: to say you _can_ do this reduction, PTAL